Package.swift

```diff
@@ -1,14 +1,6 @@
 // swift-tools-version:5.5
 import PackageDescription
 
-#if os(macOS)
-private let addCryptoSwift = false
-private let staticSwiftSyntax = true
-#else
-private let addCryptoSwift = true
-private let staticSwiftSyntax = false
-#endif
-
 #if os(Linux) && compiler(<5.6)
```

Comment:
We could probably handle this one with a combined `#if` for the compiler which + platform, but I assume it'll go away eventually anyways so we probably don't care about cross compilation for older versions